packages/ui-components/package.json

```diff
@@ -47,7 +47,6 @@
     "@rollup/plugin-commonjs": "^18.0.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-multi-entry": "^4.0.0",
-    "@svgr/rollup": "^5.5.0",
```

Comment:
A heavy package that's no longer needed if we use the svgs as components specified in js rather than .svg files